_security/configuration/index.md

```diff
@@ -3,29 +3,114 @@ layout: default
 title: Configuration
 nav_order: 2
 has_children: true
-has_toc: false
+has_toc: true
 redirect_from:
   - /security-plugin/configuration/
   - /security-plugin/configuration/index/
 ---
 
 # Security configuration
 
-The plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch in a production environment, you must configure it manually:
+The plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch in a production environment, you must configure it manually.
```

Comment:
@natebower: How does this work?

```suggestion
The Security plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch with the Security plugin in a production environment, you must make changes to the demo certificates and other configuration option manually.
```

Reply:
LGTM, but changed "option" to "options".